.taskcluster.yml

```diff
@@ -5,319 +5,308 @@ policy:
     pullRequests: public
 tasks:
     - $let:
-          taskgraph:
-              branch: taskgraph
-              revision: 81f2a1a09e9b0fc1b6c66fd452e065440793c175
           trustDomain: mobile
```

Comment:
nit: `trustDomain` is only used in one place, I've been removing the `$let` an hard-coding it below.

Could also do a full refactor in a separate PR.

Reply:
> nit: `trustDomain` is only used in one place, I've been removing the `$let` an hard-coding it below.

This sounds like it could be a footgun when copy/pasting a .tc.yml from another trust domain?

Reply:
If that's the case, I'd suggest at least moving it to the `$let` below.

Reply:
Sounds like we disagree here :)
I think this is like copying someone's bash script -- it helps if all the constants to change are at the top. I'd be in favor of moving even more variables that we need to change in-body into this section at the top, until the entire body of .tc.yml is templatized, and we only need to change the top section.

Reply:
Yeah, I like having the `trustDomain` up at the top too. It's hard to see due to the diff, but this patch is removing that first scoped `let` block and putting `trustDomain` in the same block as the Github variables. I like this because it:

A) Removes a level of indentation in almost the entire file
B) Trust domain is still at the very top
C) A comment still separates the Github vs non-Github variables (which is just trustDomain for now)